src/App.js

```diff
@@ -1,13 +1,26 @@
 // src/App.js
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import React from 'react';
 import Calculator from './components/Calculator'; // Make sure to use the correct path to Calculator.js
 import Quotes from './components/Quotes';
+import Home from './components/Home';
+import Navbar from './components/Navbar';
 
 const App = () => (
-  <div>
-    <div><Quotes /></div>
-    <Calculator />
-  </div>
+  <BrowserRouter>
+    <Navbar />
+    <Routes>
+      <Route path="/home" element={<Home />}>
```

Comment:
- [ ] [optional] @Bleu-Yves-Sopoude Great work on your app, the pages are well implemented, I would suggest you change the home router from `/home` to `/`, this will ensure your home page is not empty.